doc/manual/user-02-quickstart.md

```diff
@@ -37,9 +37,11 @@ sudo make install
 
 ## Check
 
-Make sure that `pgmoneta_ext` is installed. You should use `PostgreSQL` to test the extension functions.
+Make sure that `pgmoneta_ext` is installed. You should use `PostgreSQL` to test the extension functions. Since the extension requires the role to have superuser privileges, it is advisable to use the default `postgres` role to check if `pgmoneta_ext` has been installed successfully.
```

Comment:
The extension doesn't require `SUPERUSER` - it enables extra functionality - this need to be explained